window/WindowAsAService/Pages/Index.razor

```diff
@@ -0,0 +1,52 @@
+﻿@page "/"
+@using WindowAsAService.Models
+@using WindowAsAService.Services
+
+<h1>Hello, world!</h1>
+
+Welcome to your new app.
+<br />
+
+WindowTitle:
+<TelerikTextBox @bind-Value="@WindowTitle"></TelerikTextBox>
+<br />
+
+WindowContent
+<TelerikTextBox @bind-Value="@WindowContent"></TelerikTextBox>
+
+<br />
+<TelerikButton OnClick="@OpenNewWindow" ThemeColor="@(ThemeConstants.Button.ThemeColor.Primary)">Open New Window</TelerikButton>
```

Comment:
What about `Width` and `Height` ? They exist in the WindowBuilder as well.

In addition, the form can be polished a bit - e.g. no need for 100% wide textboxes.